Correct Nip-46 kind for nostr-connect

vazw opened the pull request.

hzrd149 commented:
I think this is right, It looks like a few months back NDK and nsecbunkerd removed the kind `24134` for the `create_account` request https://github.com/nostr-dev-kit/ndk/commit/5f9723b5f1f4001674319e22a00075d214c069cd

I updated this PR to merge into `next` and fixed the merge conflict

hzrd149 commented on src/services/nostr-connect.ts:
```diff
@@ -189,7 +189,7 @@ export class NostrConnectClient {
     method: T,
     params: RequestParams[T],
-    kind = 24134,
+    kind = 24133,
```
```suggestion
    kind = kinds.NostrConnect,
```

vazw commented:
> I think this is right, It looks like a few months back NDK and nsecbunkerd removed the kind `24134` for the `create_account` request https://github.com/nostr-dev-kit/ndk/commit/5f9723b5f1f4001674319e22a00075d214c069cd
> 
> I updated this PR to merge into `next` and fixed the merge conflict

That might be the case then🫂🫂
